src/main/java/bot/commands/audio/utils/YouTubeUtils.java

```diff
@@ -104,6 +104,9 @@ static AudioTrack getRelatedVideo(String videoID, List<AudioTrack> history) thro
         search.setKey(GetSystemEnvironmentOrDefaultValue.get("YOUTUBE_API_KEY")); //get apikey
         // need to review https://developers.google.com/youtube/v3/docs/search/list#type
         // ?????????????????????????????????? how????????
+        // after find need to add some logs
+        String id = null; //null is temp
```

Comment:
string
The relatedToVideoId parameter retrieves a list of videos that are related to the video that the parameter value identifies. The parameter value must be set to a YouTube video ID and, if you are using this parameter, the type parameter must be set to video.

Note that if the relatedToVideoId parameter is set, the only other supported parameters are part, maxResults, pageToken, regionCode, relevanceLanguage, safeSearch, type (which must be set to video), and fields.